test/functional/cmdLineTests/J9security/j9SecurityTest.xml

```diff
@@ -72,4 +72,12 @@
 		<output type="failure" caseSensitive="yes" regex="no">Processing dump event</output>
 		<output type="failure" caseSensitive="yes" regex="no">FAILED:</output>
 	</test>
+	
+	<test id="testOpenJ9GPUPermissions">
+		<command>$EXE$ $SECURITYMANAGER$ $CLASSPATH$ com.ibm.j9.security.TestOpenJ9GPUPermissions</command>
```

Comment:
I expect this should enable gpu support for full testing `-Dcom.ibm.gpu.enable=all` or `-Dcom.ibm.gpu.enable=sort`

Reply:
`-Dcom.ibm.gpu.enable=all` enables `-Dcom.ibm.gpu.enable=sort` hence adding `-Dcom.ibm.gpu.enable=all` to the command line option.

Note: this option requires permissions from module `openj9.cuda`, updated the extension PRs accordingly.
Please have another look.